pkg/northbound/utils/setconfig.go

```diff
@@ -35,23 +35,20 @@ import (
 // SetConfigAlreadyApplied is a string constant for "Already applied:"
 const SetConfigAlreadyApplied = "Already applied:"
 
-// FIXME: temporary flag to enable new plugin validation - setting to true will presently break unit tests
-var testNewPlugin = false
-
 // ValidateNetworkConfig validates the given updates and deletes, according to the path on the configuration
 // for the specified target (Atomix Based)
 func ValidateNetworkConfig(deviceName devicetype.ID, version devicetype.Version,
 	deviceType devicetype.Type, updates devicechange.TypedValueMap,
 	deletes []string, lastWrite networkchange.Revision,
 	modelRegistry *modelregistry.ModelRegistry,
 	pluginRegistry *pluginregistry.PluginRegistry,
-	deviceStateStore state.Store, allowUnvalidatedConfig bool) error {
+	deviceStateStore state.Store, allowUnvalidatedConfig bool, usePluginRegistry bool) error {
 
 	modelName := utils.ToModelName(deviceType, version)
 
 	// TODO: permanently enable the new config plugin lookup when ready
 	var configPlugin *pluginregistry.ModelPlugin
-	if testNewPlugin {
+	if usePluginRegistry {
```

Comment:
We are going to remove this hopefully next week anyway but if you need it, go for it

Reply:
yes, this will be removed once we fully migrate to the new plugin registry, but in the meantime we won't have to rebuild the `onos-config` container to test them (that was the idea when discussing it with @tomikazi )